ocaml/idl/datamodel_host.ml

```diff
@@ -1727,6 +1727,16 @@ let copy_primary_host_certs =
       ]
     ~allowed_roles:_R_POOL_ADMIN ~hide_from_docs:true ~pool_internal:true ()
 
+let set_https_only =
+  call ~name:"set_https_only" ~doc:"updates the host firewall to open or close port 80 depending on the value" 
+  ~lifecycle:[]
+  ~params:
+  [
+      (Ref _host, "host", "the host that the https_only setting is to be set")
+    ; (Bool, "value", "host.https_only will be updated according to this value")
```

Comment:
I think the doc-strings could be less self-referential: "the host that the https_only setting is to be set" -> "host whose firewall port settings to update"; "host.https_only will be updated according to this value" -> "Boolean for https_only; true - http port 80 will be blocked, false - http port 80 will be open".